extensions/kafka-connect-jdbc/README.md

```diff
@@ -0,0 +1,35 @@
+# Kafka Connect JDBC Connector
+
+kafka-connect-jdbc is a [Kafka Connector](http://kafka.apache.org/documentation.html#connect)
```

Comment:
it a fork of https://github.com/confluentinc/kafka-connect-jdbc, why not just forked a repo ?